pkg/types/trade_stat.go

```diff
@@ -0,0 +1,151 @@
+package types
+
+import (
+	"math"
+
+	"gonum.org/v1/gonum/stat"
+
+	"github.com/c9s/bbgo/pkg/datatype/floats"
+	"github.com/c9s/bbgo/pkg/fixedpoint"
+)
+
+const (
+	// DailyToAnnualFactor is the factor to scale daily observations to annual.
+	// Commonly defined as the number of public market trading days in a year.
+	DailyToAnnualFactor = 252 // todo does this apply to crypto at all?
```

Comment:
252 is an average number of trading days per year in stock market, see this link https://quant.stackexchange.com/questions/51677/why-multiply-stock-returns-with-sqrt252

I think crypto market live 365 days, no off days, so that DailyToAnnualFactor here must be 365